library/db/src/main/java/org/keynote/godtools/android/db/Contract.kt

```diff
@@ -186,7 +186,7 @@ object Contract : BaseContract() {
 
         val FIELD_ID = TABLE.field(COLUMN_ID)
         val FIELD_TOOL = TABLE.field(COLUMN_TOOL)
-        internal val FIELD_LANGUAGE = TABLE.field(COLUMN_LANGUAGE)
+        val FIELD_LANGUAGE = TABLE.field(COLUMN_LANGUAGE)
```

Comment:
Internal just means it's only available in this file right?

Reply:
`internal` means only available in this module. so previously you could only use `FIELD_LANGUAGE` from within the db module, and not reference it from `initial-content`. I removed `internal` because I'm now using this constant in a query in `initial-content`